addons/l10n_ch/models/account_invoice.py

```diff
@@ -31,31 +32,28 @@ class AccountMove(models.Model):
     @api.depends('invoice_partner_bank_id.l10n_ch_isr_subscription_eur', 'invoice_partner_bank_id.l10n_ch_isr_subscription_chf')
     def _compute_l10n_ch_isr_subscription(self):
         """ Computes the ISR subscription identifying your company or the bank that allows to generate ISR. And formats it accordingly"""
-        def _format_isr_subscription(isr_subscription):
-            #format the isr as per specifications
-            currency_code = isr_subscription[:2]
-            middle_part = isr_subscription[2:-1]
-            trailing_cipher = isr_subscription[-1]
-            middle_part = re.sub('^0*', '', middle_part)
-            return currency_code + '-' + middle_part + '-' + trailing_cipher
 
         def _format_isr_subscription_scanline(isr_subscription):
             # format the isr for scanline
             return isr_subscription[:2] + isr_subscription[2:-1].rjust(6, '0') + isr_subscription[-1:]
 
         for record in self:
+            isr_subscription = False
+            isr_subscription_formatted = False
             if record.invoice_partner_bank_id:
                 if record.currency_id.name == 'EUR':
                     isr_subscription = record.invoice_partner_bank_id.l10n_ch_isr_subscription_eur
                 elif record.currency_id.name == 'CHF':
                     isr_subscription = record.invoice_partner_bank_id.l10n_ch_isr_subscription_chf
                 else:
                     #we don't format if in another currency as EUR or CHF
-                    continue
+                    pass
```

Comment:
better to remove this

Reply:
was keeping it for the logic and the comment.

I'll remove it.